llvm/utils/git/code-format-helper.py

```diff
@@ -44,6 +44,7 @@ class FormatArgs:
     token: str = None
     verbose: bool = True
     issue_number: int = 0
+    write_comment_to_file: bool = False
```

Comment:
You could in theory run the command locally and have it comment on the PR, which is why I left it in there.  I don't know if this is a feature people want to have though.

Reply:
Given that the CI exists, I think that functionality would be pretty redundant. The only reason I've run it locally in the past and had it comment on a PR was for testing, which should be quite a bit easier now that it spits everything out into a file rather than needing to comment directly.

Reply:
Yeah I think we can remove the dead code, but we can do that in a new commit after this. I don't see any reason to have that in there, we could also just change the code to always write to comment file without the arg.  The change of functionality needs to be documented in the comment of the file, so that if we wan to set this up somewhere else we know what's expected.